$NetBSD: patch-madspinh,v 1.1 2007/05/23 11:51:54 rillig Exp $

gcc4 says:
config.c:34: error: static declaration of 'config_window' follows non-static declaration
madspin.h:11: error: previous declaration of 'config_window' was here

--- madspin.h.orig	2003-02-20 21:20:32.000000000 +0100
+++ madspin.h	2007-05-23 13:49:31.000000000 +0200
@@ -8,7 +8,6 @@ typedef struct
 madspin_config;
 
 /* config.c */
-extern GtkWidget *config_window;
 extern madspin_config config;
 extern void madspin_configure (void);
 extern void config_read (void);
